packages/dashboard/src/components/composed/RPCs/RPC/Overview.tsx

```diff
@@ -0,0 +1,140 @@
+  return {
+    container: {
+      backgroundColor:
+        colorScheme === "dark"
+          ? fn.rgba(colors["truffle-beige"][8], 0.12)
+          : colors["truffle-beige"][1],
+      transition: "background-color 0.2s",
+      borderRadius: `${radius.sm}px ${radius.sm}px 0 0`,
+      cursor: "pointer"
+    },
+    activeContainer: {
+      backgroundColor:
+        colorScheme === "dark"
+          ? fn.rgba(colors["truffle-beige"][8], 0.2)
+          : white,
+      [`& .${getRef("button")}`]: {
+        opacity: 1
+      },
+      [`& .${getRef("rejectButton")}`]: {
+        "backgroundColor": fn.rgba(colors.red[8], 0.5),
+        "&:hover": {
+          backgroundColor: colors.red[8]
+        }
+      },
+      [`& .${getRef("confirmButton")}`]: {
+        "backgroundColor": fn.rgba(colors["truffle-teal"][9], 0.5),
+        "&:hover": {
+          backgroundColor: colors["truffle-teal"][9]
+        }
+      }
+    },
+    methodName: {
+      fontFamily: fontFamilyMonospace,
+      fontWeight: 700,
+      color:
+        colorScheme === "dark"
+          ? colors["truffle-beige"][3]
+          : colors["truffle-beige"][8]
+    },
+    button: {
+      ref: getRef("button"),
+      backgroundColor: colors.gray[7],
+      transition: "all 0.2s",
+      opacity: 0.15
+    },
+    rejectButton: { ref: getRef("rejectButton") },
+    confirmButton: { ref: getRef("confirmButton") },
+    confirmButtonRightIcon: {
+      marginLeft: 4,
+      marginRight: 6
+    }
+  };
+});
+
+type OverviewProps = {
+  lifecycle: ReceivedMessageLifecycle<DashboardProviderMessage>;
+  active: boolean;
+  onBackClick: React.MouseEventHandler<HTMLDivElement>;
+  onBackEnter: React.MouseEventHandler<HTMLDivElement>;
+  onBackLeave: React.MouseEventHandler<HTMLDivElement>;
+  onRejectButtonEnter: React.MouseEventHandler<HTMLButtonElement>;
+  onRejectButtonLeave: React.MouseEventHandler<HTMLButtonElement>;
+  onConfirmButtonEnter: React.MouseEventHandler<HTMLButtonElement>;
+  onConfirmButtonLeave: React.MouseEventHandler<HTMLButtonElement>;
+};
+
+function Overview({
+  lifecycle,
+  active,
+  onBackClick,
+  onBackEnter,
+  onBackLeave,
+  onRejectButtonEnter,
+  onRejectButtonLeave,
+  onConfirmButtonEnter,
+  onConfirmButtonLeave
+}: OverviewProps): JSX.Element {
+  const { method } = lifecycle.message.payload;
+  const {
+    state: { chainInfo },
+    operations: { userConfirmMessage, userRejectMessage }
+  } = useDash()!;
+  const { classes } = useStyles();
+
+  const onConfirmButtonClick = () => void userConfirmMessage(lifecycle);
+  const onRejectButtonClick = () => void userRejectMessage(lifecycle);
+
+  return (
+    <Group
+      onClick={onBackClick}
+      onMouseEnter={onBackEnter}
+      onMouseLeave={onBackLeave}
+      position="apart"
+      spacing={50}
+      pl={42}
+      pr={35}
+      py="lg"
+      className={`${classes.container} ${
+        active ? classes.activeContainer : ""
+      }`}
+      tabIndex={0}
+    >
```

Comment:
I really dig the signifiers you have for expanding / rejecting confirming! The mouse-over effect suggesting "click to expand" looks beautiful and clearly guides usage, and the accept/reject check/x are nice touches :)

I noticed however that this disappears when expanded - confirm/reject don't show up anymore, and it'd be great to have the reverse chevrons-up to show I can collapse the view.

<details>
<summary>See what I mean</summary>

![the-signifiers-duke-the-signifiers](https://user-images.githubusercontent.com/151065/183800005-70c5adcf-6599-4840-8510-c5235c6823c1.gif)

</details>

Reply:
Yea I omitted it because I thought it'd make the expanded view cleaner, but I think this makes sense! I've [made the change](https://github.com/trufflesuite/truffle/pull/5241/commits/2d94c4775b82d70de0bd00bdbccb9423d0a370a8), still clean :)

Reply:
Amazing